src/lib.rs

```diff
@@ -256,6 +256,22 @@
 //! env_logger::Builder::from_env(Env::default().default_filter_or("warn")).init();
 //! ```
 //!
+//! ## Using the Logger as a Timer
+//!
+//! `env_logger` can also be used as a timer by moving an initial
+//! timestamp into the closure defining a custom format:
+//!
+//! ```
+//! use std::io::Write;
+//!
+//! let start = std::time::Instant::now();
+//! env_logger::builder()
+//!     .format(move |buf, record| {
+//!         writeln!(buf, "{} ({}): {}", record.level(), start.elapsed().as_millis(), record.args())
+//!     })
+//!     .init();
+//! ```
+//!
```

Comment:
I was assuming this would be created as another example in `examples/`.  The advantage of this is its easier to interactively play with to see the results.

If its being added to the docs, putting it on [`Builder::format`](https://docs.rs/env_logger/latest/env_logger/struct.Builder.html#method.format) might be a better approach.

Reply:
So sorry to disappoint your expectations 😆.